cmd/skopeo/copy.go

```diff
@@ -87,6 +86,10 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) error {
 		return errorShouldDisplayUsage{errors.New("Exactly two arguments expected")}
 	}
 
+	if err := needReexec(args); err != nil {
```

Comment:
This works, but it’s still non-obvious that this is tied with the `args[…]` uses elsewhere. Please use actual variables,  like in https://github.com/containers/skopeo/pull/653#discussion_r284781828 , so that the `args`→image name “semantic conversion” happens at exactly one place. (IOW, so that we can, e.g. rip out and replace the CLI framework without changing the logic at all.)

Reply:
Done